### PR TITLE
error message in assertion updated for resource share test

### PR DESCRIPTION
**Pull request by S-Panta**
Related issue : https://github.com/owncloud/ocis-php-sdk/issues/131

### CODE REVIEW

File: tests/integration/Owncloud/OcisPhpSdk/ResourceInviteTest.php [hunk 1]
```diff
@@ -124,12 +162,38 @@ public function testInviteWithExpiry(): void
...
         $share = $this->fileToShare->invite($this->einstein, $this->viewerRole, $tomorrow);
-        $this->assertInstanceOf(\DateTimeImmutable::class, $share->getExpiration());
-        $this->assertSame($tomorrow->getTimestamp(), $share->getExpiration()->getTimestamp());
+        $shareResourceExpirationDateTime = $share->getExpiration();
+        $this->assertInstanceOf(
+            \DateTimeImmutable::class,
+            $shareResourceExpirationDateTime,
+            "Expected class to be " . \DateTimeImmutable::class
+            . " but found " . ($shareResourceExpirationDateTime !== null ? get_class($shareResourceExpirationDateTime) : "null")
+        );
```
Comment (individual-it) [inline tests/integration/Owncloud/OcisPhpSdk/ResourceInviteTest.php:171]: what about simplifying it like that

```suggestion
        $this->assertInstanceOf(
            \DateTimeImmutable::class,
            $shareResourceExpirationDateTime,
            "Expected class to be 'DateTimeImmutable'"
            . " but found " . print_r($shareResourceExpirationDateTime, true)
        );
```

**Review by individual-it — CHANGES_REQUESTED** [review]

File: tests/integration/Owncloud/OcisPhpSdk/ResourceInviteTest.php (hunk 1 above)
Comment (individual-it) [reply tests/integration/Owncloud/OcisPhpSdk/ResourceInviteTest.php]: there are more cases like that, e.g. https://github.com/owncloud/ocis-php-sdk/pull/186/files#diff-63ac34bf34e0074dd6979a9dd8623738a73f3d9f457baee479282eb29c1353acR156-R161

File: tests/integration/Owncloud/OcisPhpSdk/ResourceInviteTest.php [hunk 2]
```diff
@@ -224,10 +348,15 @@ public function testInviteUserToAReceivedShare(): void
          * @var ShareReceived $receivedShare
          */
         $receivedShare = $this->getSharedWithMeWaitTillShareIsAccepted($this->marieOcis)[0];
-        $this->assertSame('to-share-test.txt', $receivedShare->getName());
+        $this->assertSame(
+            'to-share-test.txt',
+            $receivedShare->getName(),
+            "Expected shared resource to be 'to-share-test.txt but found " . $receivedShare->getName()
```
Comment (amrita-shrestha) [inline tests/integration/Owncloud/OcisPhpSdk/ResourceInviteTest.php:354]:
```suggestion
            "Expected shared resource to be 'to-share-test.txt' but found " . $receivedShare->getName()
```

**Review by amrita-shrestha — APPROVED** [review]
lgtm

File: tests/integration/Owncloud/OcisPhpSdk/ResourceInviteTest.php [hunk 3]
```diff
@@ -143,16 +207,48 @@ public function testInviteWithExpiryTimezone(): void
...
+            "Expected class to be 'DateTimeImmutable' but found "
+            . print_r($shareResourceExpirationDateTime, true)
+        );
         // The returned expiry is in UTC timezone (2 hours earlier than the expiry time in Kyiv)
-        $this->assertSame("Thu, 01 Jan 2060 10:00:00 +0000", $share->getExpiration()->format('r'));
-        $this->assertSame("Z", $share->getExpiration()->getTimezone()->getName());
+        $this->assertSame(
+            "Thu, 01 Jan 2060 10:00:00 +0000",
+            $shareResourceExpirationDateTime->format('r'),
+            "Expected expiration datetime of shared resource doesn't match"
```
[the remark this answers is not in the document]
Comment (amrita-shrestha) [reply tests/integration/Owncloud/OcisPhpSdk/ResourceInviteTest.php:221]: @nirajacharya2 @S-Panta

[the remark this answers is not in the document]
Comment (S-Panta) [reply tests/integration/Owncloud/OcisPhpSdk/ResourceInviteTest.php:221]: Expiration has both a date and time into it. SO, it's better not to change that imo.

**Pull request merged**